GSA/digital-gov-drupal
# Digital-239: Custom 12 hour logout

## Jira ticket




[DIGITAL-239](https://cm-jira.usa.gov/browse/DIGITAL-239)

## Purpose


This PR will create a custom module to add the 12 hour log out timer for logging users out

## Includes the following PRs that must be merged first



n/a

## Deployment and testing

### Local Setup



`lando rebuild -y && lando si`

### QA/Testing instructions



Visit `/admin/config/people/dg_autologout` and verify that the time is set to 12 hours and will log a user out no matter role or use

## Checklist for the Developer




## Checklist for the Peer Reviewers
